charts/flyte/templates/admin/flytescheduler.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.flytescheduler.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "flytescheduler.name" . }}
+  namespace: {{ template "flyte.namespace" . }}
+  labels: {{ include "flytescheduler.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.flytescheduler.replicaCount }}
+  selector:
+    matchLabels: {{ include "flytescheduler.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        configChecksum: {{ include (print .Template.BasePath "/admin/configmap.yaml") . | sha256sum | trunc 63 | quote }}
+        {{- with .Values.flytescheduler.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels: {{ include "flytescheduler.labels" . | nindent 8 }}
+    spec:
+      initContainers:
+      - command: ["/bin/sleep","60"]
+        image: "alpine:3.3"
+        imagePullPolicy: "{{ .Values.flytescheduler.image.pullPolicy }}"
+        name: sleep
+      containers:
+      - command:
```

Comment:
Can you add `resources` section? I can see it in the values.yaml, but it's not in the template.